corehq/messaging/scheduling/static/scheduling/js/broadcasts_list.js

```diff
@@ -104,9 +109,10 @@ hqDefine("scheduling/js/broadcasts_list", function() {
         });
     });
 
-    function activateDeactivate(action, broadcast_id) {
-        var button_id = 'activate-button-for-scheduled-broadcast-' + broadcast_id;
-        $('#' + button_id).prop('disabled', true);
+    function broadcastAction(action, broadcast_id) {
+        $('#activate-button-for-scheduled-broadcast-' + broadcast_id).prop('disabled', true);
```

Comment:
Minor ux: you can call `$element.disableButton()` to disable and add a spinner, which is a nice explicit indicator that something is processing.

Reply:
cool, https://github.com/dimagi/commcare-hq/pull/19027/commits/af4357ebbb247eb47eb720b31703d9f67e9a39dc